Styles/Style.css

```
@@ -0,0 +1,420 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+header{
+    font-family: 'Merriweather', serif;
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: linear-gradient(to right, #4845E4, #5722CD);
+    color: #fff;
+    padding: 1rem 1rem 10rem 1rem;
+    text-align: center;
+    position: relative;
+}
+
+header:after{
+    content: '';
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    background: inherit;
+    z-index: -1;
+    bottom: 0;
+    transform-origin: left bottom;
+    transform: skewY(2deg);
+}
+
+
+header nav {
+    display: flex;
+    align-self: flex-end;
+    margin-right: 10rem;
+}
+
+nav ul {
+    display: flex;
+    gap: 2rem;
+}
+
+ul a {
+    color: #ffffff;
+    text-decoration: none;
+    font-size: 0.8rem;
+}
+
+ul li{
+    list-style-type: none;
+}
+
+.content{
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    flex: 1 1 15rem;
+    flex-wrap: wrap;
+    width: 30%;
+}
+
+.content p{
+    font-size: 0.8rem;
+    margin: 1rem 0;
+}
+
+
+header button {
+    background-color: #ED4385;
+    color: #ffffff;
+    padding: 0.8rem 1.3rem;
+    border-radius: 2rem;
+    text-transform: uppercase;
+    font-size: 0.6rem;
+    margin-top: 1rem;
+}
+
+.squiggle {
+    background-image: url(/Images/squiggle_line_stripe.svg);
+    width: 5rem;
+    height: 5rem;
+    position: absolute;
+    bottom: -2.2rem;
+}
+
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.introduction {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 40%;
+    white-space: pre-line;
+    flex: 1 1 10rem;
+    gap: 3rem;
+    padding: 10rem 0 25rem 0;
+    position: relative;
+}
+
+.hello-section:after {
+    content: "hello";
+    font-size: 23rem;
+    font-weight: 500;
+    position: absolute;
+    color: rgb(242, 240, 255);
+    top: 0;
+    z-index: -1;
+}
+
+.introduction h3 {
+    text-align: center;
+    color: #5722CD;
+}
+
+.introduction a {
+    text-decoration: none;
+    color: #5722CD;
+}
+
+/* --------------- 4.3 Main --------------------------- */
+.container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 4rem;
+    padding: 2rem;
+    position: relative;
+}
+
+.outer-container {
+    display: flex;
+    gap: 20rem;
+    flex: 1 3 auto;
+}
```

```suggestion
    align-items: center;
}
```

Hiermee staat de afbeelding en tekst horizontaal gecentreerd